nx-libs.spec

```diff
@@ -357,6 +357,10 @@ find -name LICENSE | xargs sed -i \
   -e 's/MA  02111-1307/MA  02110-1301/'
 # Fix source permissions
 find -type f -name '*.[hc]' | xargs chmod -x
+# RHEL6 and lower do not know -Wpedantic
+%if 0%{rhel} < 7
```

Comment:
Ok, I have reworked the patch. It is building fine once I remove the other gcc from the PATH... Please have a second glance.